miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditRunController.java

```diff
@@ -209,7 +215,8 @@ public ModelAndView setupForm(Run run, PlatformType platformType, ModelMap model
         model.put("title", "Run " + run.getId());
         model.put("multiplexed", isMultiplexed(run));
         model.put("metrics",
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor the code so this stream pipeline is used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3958)